dashboard/app/models/levels/music.rb

```diff
@@ -63,4 +67,31 @@ def enable_scrolling?
     # ensures we have the small footer
     true
   end
+
+  # Ensure that if this is a multiple choice predict level, there is at least one correct answer
```

Comment:
I wonder if we should extract these to a helper file since we are reusing them in python lab and music lab, and will likely re-use them again.

Reply:
+1, or move them into `level.rb`?